doc/architecture/decisions/0005-persist-submitted-documents-in-translated-json.md

```diff
@@ -0,0 +1,44 @@
+# 5. Persist Submitted Documents in Translated JSON
+
+Date: 2022-10-17
+
+## Status
+
+Accepted
+
+## Context
+
+We need to ensure that submitted and assessed Applications are
+reliably persisted as permanent read-only artefacts. We know that they will
+have an enduring role as a snapshot of an individual's AP-related risks and
+needs at a point in time. The application, along with the content added during
+the assessment stage will be an important reference document at every stage
+in the lifecycle of person's relationship with the probation service.
+
+Throughout the lifecycle of the product, it is expected that the questions will
+change, so it is important that we ensure that any changes made to the questions
+still mean that older Applications are still readable.
```

Comment:
I suggested adding: 'still readable with the questions as they were when the application was completed'

Reply:
👍